src/server/worldserver/worldserver.conf.dist

```diff
@@ -4034,6 +4035,24 @@ PacketSpoof.BanMode = 0
 
 PacketSpoof.BanDuration = 86400
 
+#
+###################################################################################################
+
+###################################################################################################
+# MISC ANTI-CHEAT SETTINGS
+#
+# Settings to tweak various anti cheat measures.
+#
+#    AntiCheats.PendingMoveChangesTimeoutTime
+#        Description: Time each client has to acknowledge a movement change until they are kicked.
+#        Note:        A lower time will give cheaters less advantage when using a very specific
+#                     type of hack. However, the cost is that innocent but laggy players might
+#                     sometimes get kicked.
+#        Default:     2000 (2 seconds)
+
+AntiCheats.PendingMoveChangesTimeoutTime = 2000
```

Comment:
I think it's more consistent to not use plural here as we don't count anti-cheat as an entity but as a type of mechanic

Reply:
Done